source/auth/auth.rst

```diff
@@ -6,14 +6,14 @@ Driver Authentication
 =====================
 
 :Spec: 100
-:Spec Version: 1.9.1
+:Spec Version: 1.9.2
```

Comment:
This is worth a bump of the minor version

Reply:
Nit: it should be 1.10.0

Reply:
Updated.